### DISCUSSION TITLE
Allow disabling autocomplete on an input field

### DISCUSSION THREAD

**Discussion by Semvrij** [tailwindlabs/headlessui#1310] [Ideas]
I want to disable autocomplete on the ComboboxInput.

[How to turn off form autocompletion
](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion)

**Comment by uglow** [comment]
The docs [hint](https://headlessui.dev/react/combobox#rendering-a-different-element-for-a-component) at how you can do this. This worked for me: 

```
<Combobox.Input as={Fragment} >
   <input autoComplete="off"/>
</Combobox.Input>
```

**Reply by RobinMalfait** [reply]
You should also be able to put it directly in the Input component:
```js
<Combobox.Input autoComplete="off" />
```
